hail/python/hailtop/utils/utils.py

```diff
@@ -26,7 +26,7 @@ def async_to_blocking(coro):
         thread_pool, lambda: fun(*args, **kwargs))
 
 
-class AsyncWorkerPool:
+class AsyncThrottledGather:
```

Comment:
Why would we prefer this class to the one with an explicit queue? (other than the change to exception handling)

Reply:
I don't think it matters too much. This version has a bunch of futures created, so maybe it would be better to have an explicit queue.